### PR TITLE
Extending: Exercise 3

### PR DESCRIPTION
**Pull request by mahoneycm**
# Summary

- Creates and uses `testimonial` include
- Creates and forwards `ut-testimonial` styles

### CODE REVIEW

**Review by mejiaj — CHANGES_REQUESTED** [review]

File: assets/css/components/ut-testimonial.scss [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+.ut-testimonial__profile {
+  margin-left: 1rem;
+}
+
+.ut-testimonial__profile-name {
+  @include u-text("indigo-10v");
+}
+
+.ut-testimonial__profile-title {
+  font-size: 1rem;
```
[the remark this answers is not in the document]
Comment (mahoneycm) [reply assets/css/components/ut-testimonial.scss]: Resolved in [1b4c780](https://github.com/uswds/uswds-tutorial/pull/31/commits/1b4c7802f2c377d70f7358b42fdac159ec27626a)

File: assets/css/components/ut-testimonial.scss [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  flex-direction: row;
+}
+
+.ut-testimonial img {
+  height: 75px;
+  width: 75px;
+}
+
+.ut-testimonial__profile {
+  margin-left: 1rem;
```
[the remark this answers is not in the document]
Comment (mahoneycm) [reply assets/css/components/ut-testimonial.scss]: Resolved in [1b4c780](https://github.com/uswds/uswds-tutorial/pull/31/commits/1b4c7802f2c377d70f7358b42fdac159ec27626a)

File: assets/css/components/ut-testimonial.scss [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+}
+
+.ut-testimonial__footer {
+  display: flex;
+  flex-direction: row;
+}
+
+.ut-testimonial img {
+  height: 75px;
+  width: 75px;
```
[the remark this answers is not in the document]
Comment (mahoneycm) [reply assets/css/components/ut-testimonial.scss]: Resolved in [1b4c780](https://github.com/uswds/uswds-tutorial/pull/31/commits/1b4c7802f2c377d70f7358b42fdac159ec27626a)

File: _includes/sxs/02-extending/testimonial.html [hunk 4]
```diff
@@ -0,0 +1,19 @@
+<div class="usa-card position-sticky top-1 ut-testimonial">
```
[the remark this answers is not in the document]
Comment (mahoneycm) [reply _includes/sxs/02-extending/testimonial.html]: Resolved in [5c284e7](https://github.com/uswds/uswds-tutorial/pull/31/commits/5c284e77ecd0c85523390bc638eea3f4b40f66d1)

**Review by mejiaj — APPROVED** [review]
Looks great!

**Pull request merged**